src/components/Canvas.vue

```diff
@@ -37,12 +37,27 @@
       />
       <div
         class="absolute border-2 border-solid border-red-800"
-        :style="`width:${curw}px; height:${curh}px;
-            left: ${moveToolPos.x}px; top: ${moveToolPos.y}px;`"
+        :style="
+          `width:${curw}px; height:${curh}px; ` +
+          `left: ${moveToolPos.x - curw / 2}px; ` +
```

Comment:
Please use computed.

Reply:
Nice call! I actually intended to store the center of the moving tool instead of storing the point that actually starts rendering since the center of the moving tool point will use within the logic, but still better to store the value which offset with the cursor tool width.